packages/react-scripts/template/README.md

```diff
@@ -1063,8 +1068,9 @@ and then use them in your tests like you normally do.
 ### Initializing Test Environment
 
 >Note: this feature is available with `react-scripts@0.4.0` and higher.
+>Note: After ejecting you need to manually add [setupTestFrameworkScriptFile](https://facebook.github.io/jest/docs/configuration.html#setuptestframeworkscriptfile-string) in package.json. 
```

Comment:
This doesn't render very nicely. Please use GH Preview feature to test your changes. Thanks!
<img width="1130" alt="screen shot 2017-06-27 at 5 28 05 pm" src="https://user-images.githubusercontent.com/810438/27598645-0aba8ac2-5b5e-11e7-8bdb-3c3a746f7ddb.png">